src/engine_registry.cpp

```diff
@@ -31,11 +31,11 @@ namespace wr
 		);
 
 	REGISTER(root_signatures::brdf_lut, RootSignatureRegistry)({
-		RootSignatureDescription::Parameters({
+		.m_parameters = {
```

Comment:
Not entirely sure what the . prefix does here, same goes for the other prefixes, it is explicit and clear though but might be confusing for the team